RELEASE_PROCESS.md

```diff
@@ -0,0 +1,41 @@
+## Definition
+
+A release is mostly a git tag of http://github.com/wallabag/wallabag, following [semantic versioning](http://semver.org).
+The last release at the time of writing is 2.0.0-alpha.2, from the v2 branch.
+
+### TL;DR
+- create a branch for the release, from `v2`: `git checkout -b release-2.0.0-alpha.3`
+- run composer update to make sure `composer.lock` is up-to-date
```

Comment:
yes, I think we can do your command, @j0k3r. For next alpha, I just did `composer update`.